src/test/java/org/openrewrite/java/logging/PrintStackTraceToLogErrorTest.java

```diff
@@ -235,4 +236,45 @@ public void close() {
           )
         );
     }
+
+    @Test
+    @Issue("https://github.com/openrewrite/rewrite-logging-frameworks/issues/114")
+    void supportLombokLogAnnotations() {
+        rewriteRun(
+          spec -> spec.recipe(new PrintStackTraceToLogError(null, null, null))
+            .parser(JavaParser.fromJavaVersion().classpath("slf4j-api", "lombok"))
+            .typeValidationOptions(TypeValidation.builder().identifiers(false).build()),
```

Comment:
Since the fields are absent, we have missing type information in the result. I figured that's ok & to be expected.

Reply:
No we don't run the Lombok annotation processor, so we don't have that information available; it's good enough to make this code change; not quite sure if any followup recipes would work, but the JavaTemplates we use should have the type information, as per
https://github.com/openrewrite/rewrite-logging-frameworks/blob/53f87871ef81a07e03ca4e1dc0577aaf2117fea0/src/main/java/org/openrewrite/java/logging/SystemErrToLogging.java#L178-L185